### ISSUE TITLE
Allow re-rendering after config has changed

### ISSUE THREAD

**Issue by jamchamb** [rf/renderizer#7]
After changing the config we'd want to render our images with new settings, but if the script sees that the source images haven't changed it refuses to re-render. Could check to see if `images.yaml` is newer than the last outputs.

**Comment by rf** [comment]
Yeah the way I used to get around this was to just `find images/ | xargs touch`. `touch` will just set their last modified date to the current time, so that'll force rebuild. But yeah, that would be a cool feature.

**Comment by jamchamb** [comment]
I might've jumped the gun on this one. It works, but *all* output groups will be re-rendered after the config is updated, whereas we really only want to re-render the specific groups that were updated. To do that, we'd have to keep track of the properties used to generate each output file.

It might be possible to use a flat-file to store hashes of each `outputConfig` by group and do a re-render if we come across an `outputConfig` that hasn't been seen before (and then prune the ones that no longer exist) in addition to re-rendering when the source images change.

In the meantime, should I undo the changes to `ModCheck`? I could also just put a `--rerender <group(s)>` option in its place until something more automatic is implemented.

**Comment by jamchamb** [comment]
I've been using a simple modification that just puts a hash of the output description into the filename of the build files, and it works pretty well (I think the 9patch plugin has to be updated to work with it, though). The only issue is leftover obsolete build images, but it shouldn't be hard to add a command line option for pruning out configs that no longer exist in `images.yaml`.
